oap-server/server-starter/src/main/resources/lal/mysql-slowsql.yaml

```diff
@@ -0,0 +1,34 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+rules:
+  - name: mysql-slowsql
+    dsl: |
+      filter {
```

Comment:
I think we should set `layer=Layer.MYSQL` condition in this rule. 

And this file is not added into `log-analyzer/default/lalFiles` to activate.

Reply:
Also add `layer=Layer.MESH` condition in `envoy-als.yaml` rule. And we should activate that rule in application.yml too. @kezhenxu94 Is this correct?

Reply:
> Also add `layer=Layer.MESH` condition in `envoy-als.yaml` rule.

We can add this.

> And we should activate that rule in application.yml too. @kezhenxu94 Is this correct?

I'm afraid no, `envoy-als.yaml` should be activated manually, because there is no obvious tags that we can use to determine whether the logs are from envoy ALS or not and it's troublesome for users to add a custom tag in ALS (like `tag("LOG_KIND") == "SLOW_SQL"` what we did in the slow sql case here)

Reply:
Don't we set the layer for als?

Reply:
> Don't we set the layer for als?

OK, I just checked, we have set the layer before forwarding the ALS logs to LAL.

https://github.com/apache/skywalking/blob/526aa3a5fd077ce548eed8e78052eff394209847/oap-server/server-receiver-plugin/envoy-metrics-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/envoy/persistence/LogsPersistence.java#L92-L93

Reply:
@yswdqz Please use layer as precondition, and activate all LAL script.

Reply:
I get it.

Reply:
just like this?
```
rules:
  - name: mysql-slowsql
    dsl: |
      filter {
        if (log.layer != "MYSQL") {
          abort{} 
        }
        json{
        }
        extractor{
          layer parsed.layer as String
          service parsed.service as String
          timestamp parsed.time as String
          if (tag("LOG_KIND") == "SLOW_SQL") {
             slowSql {
               id parsed.id as String
               statement parsed.statement as String
               latency parsed.query_time as Long
             }
          }
        }
      }
```

Reply:
Yes, put layer check in all LAL scripts.